package.json

```diff
@@ -1,22 +1,26 @@
 {
-  "name": "node.js_course_practice",
-  "version": "1.0.0",
-  "description": "",
-  "main": "app.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/DimitriIlnitski/Node.js_Course_Practice.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/DimitriIlnitski/Node.js_Course_Practice/issues"
-  },
-  "homepage": "https://github.com/DimitriIlnitski/Node.js_Course_Practice#readme",
-  "dependencies": {
-    "express": "^4.18.2"
-  }
+	"name": "node.js_course_practice",
+	"version": "1.0.0",
+	"description": "",
+	"main": "app.js",
+	"scripts": {
+    "start": "node app.js",
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"generate-swagger": "node swagger.js"
```

Comment:
What does this command do? Is it needed?

Reply:
Have added by mistake. Deleted